src/test/java/org/takes/facets/auth/social/PsGoogleTest.java

```diff
@@ -104,7 +104,7 @@ public Response act(final Request req) throws IOException {
                         MatcherAssert.assertThat(
                             new RqPrint(req).printHead(),
                             Matchers.containsString(
-                                "GET /oauth2/v1/userinfo"
+                                "GET /plus/v1/people/me"
```

Comment:
@Happy-Neko we need check parameters as well
if parameter `image/url` is needed